### ISSUE TITLE
vala 0.31 on windows 64

### ISSUE THREAD

**Issue by Mparaiso** [msys2/MINGW-packages#9]
Hi, after installing base-devel, vala 0.30 , glib-devel and gettext-devel from pacman I successfully built and installed vala 0.31 from sources available on gnome website, I wonder how I could help , Is there a wiki which describe how to write a package then push it somewhere to make it available for other users ? thanks . 

In fact, I'd be willing to help push other packages I've built if someone give me an explanation.

tldr; I'm eager to contribute.
